src/main/java/com/google/devtools/build/lib/bazel/rules/python/python_stub_template.txt

```diff
@@ -106,7 +106,14 @@ def FindModuleSpace():
     if os.path.isdir(module_space):
       return module_space
 
-    runfiles_pattern = r'(.*\.runfiles)' + (r'\\' if IsWindows() else '/') + '.*'
+    slash_pattern = r'\\' if IsWindows() else '/'
+    runfiles_pattern = r'(.*\.runfiles)' + slash_pattern + '.*'
```

Comment:
This change comparing to the old implementation reduces readability in my opinion.